From 9b80b4a4ea4163f65004f2b16e65b8adcdf2b3dc Mon Sep 17 00:00:00 2001
From: Marek Haicman <mhaicman@redhat.com>
Date: Thu, 4 Jan 2018 00:29:26 +0100
Subject: [PATCH] Make command in bash role header in line with --help

Point is to make more explicit it's a placeholder name - using
the same name --help use might do the trick.
---
 src/XCCDF_POLICY/xccdf_policy_remediate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/XCCDF_POLICY/xccdf_policy_remediate.c b/src/XCCDF_POLICY/xccdf_policy_remediate.c
index fbbad8885..1a7d21ed7 100644
--- a/src/XCCDF_POLICY/xccdf_policy_remediate.c
+++ b/src/XCCDF_POLICY/xccdf_policy_remediate.c
@@ -846,7 +846,7 @@ static int _write_script_header_to_fd(struct xccdf_policy *policy, struct xccdf_
 			"# Benchmark Version:  %s\n#\n"
 			"# XCCDF Version:  %s\n#\n"
 			"# This file was generated by OpenSCAP %s using:\n"
-			"# 	$ oscap xccdf generate fix --profile %s%s%s sds.xml \n#\n"
+			"# 	$ oscap xccdf generate fix --profile %s%s%s xccdf-file.xml \n#\n"
 			"# This script is generated from an OpenSCAP profile without preliminary evaluation.\n"
 			"# It attempts to fix every selected rule, even if the system is already compliant.\n"
 			"#\n"
